src/components/SearchForm.vue

```diff
@@ -90,6 +90,10 @@ export default {
   .algolia-autocomplete {
     width: 100%;
   }
+
+  &__input {
+    padding-right: 2.3rem;
+  }
```

Comment:
I don't think we should put this. Please avoid to use random units e.g. `2.3rem` Thanks for your efforts ! @nikitamadin